### DISCUSSION TITLE
我建立本地储存的时候报错 Failed create storage in database: UNIQUE constraint failed: x_storages.mount_path

### DISCUSSION THREAD

**Discussion by duliri888** [AlistGo/alist#1818] [General]
我用docker安装的 我建立本地储存的时候报错 Failed create storage in database: UNIQUE constraint failed: x_storages.mount_path
状态:root folder /data/aria2-downloads not exists

**Comment by anwen-anyi** [comment]
https://alist.nn.ci/zh/faq/why.html#为什么我在添加新存储时收到错误-failed-create-storage-in-database-unique-constraint-failed-x-storages-mount-path

**Comment by souImate** [comment]
bug  刷新下 可以看到已经建好了

**Reply by milipapa** [reply]
3Q~~
